requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+https://github.com/nengo/nengo#egg=nengo
+git+https://github.com/nengo/nengo@learning-unmodulate#egg=nengo
```

Comment:
I worry slightly about this squash - After getting the PES branch to this
state, I branched from it to begin work on Voja and that relationship seems
to have been lost in the squash which I fear may cause git to do horrible
things when it comes to merge that branch.

On 16 June 2015 at 21:49, Andrew Mundy <notifications@github.com> wrote:

> In requirements.txt
> <https://github.com/project-rig/nengo_spinnaker/pull/29#discussion_r32569050>
> :
>
> > @@ -1,2 +1,2 @@
> > -git+https://github.com/nengo/nengo#egg=nengo
> > +git+https://github.com/nengo/nengo@learning-unmodulate#egg=nengo
>
> This will need reverting to normal before merging.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/project-rig/nengo_spinnaker/pull/29/files#r32569050>.
>